index.bs

```diff
@@ -658,6 +652,23 @@ The [=reading change threshold|threshold=] value depends on the surrounding soft
 environment constraints, e.g., software power consumption optimizations or the underlying
 [=device sensor=]'s accuracy.
 
+## Conditions to expose sensor readings ## {#concepts-can-expose-sensor-readings}
+
+The user agent can define the list of [=conditions=] to be used to check if
+<dfn lt="can expose sensor readings">sensor readings can be exposed</dfn>.
```

Comment:
I would say:

`to check if it <dfn>can expose sensor readings</dfn>`

That is, use the "can expose sensor readings" operation's canonical name.